src/tracer.js

```diff
@@ -236,9 +247,9 @@ function getTrimmedTrace(traceSize, jsTrace) {
                 }
             }
         }
-        if (totalTrimmedEvents) {
-            utils.debugLog(`Epsagon - Trace size is larger than maximum size, ${totalTrimmedEvents} events was trimmed.`);
-        }
+    }
+    if (totalTrimmedEvents || totalTrimmedExceptions) {
+        utils.debugLog(`Epsagon - Trace size is larger than maximum size, ${totalTrimmedEvents} events and ${totalTrimmedExceptions} exceptions was trimmed.`);
```

Comment:
"... were trimmed."

Reply:
Consider adding logging for trimmed metadata as well.

Reply:
There is an indication (is_trimmed) inside the events when we're trimming event metadata.